llvm/lib/Passes/StandardInstrumentations.cpp

```diff
@@ -119,8 +119,8 @@ static cl::opt<bool> PrintPassNumbers(
     cl::desc("Print pass names and their ordinals"));
 
 static cl::opt<unsigned>
-    PrintAtPassNumber("print-at-pass-number", cl::init(0), cl::Hidden,
-                cl::desc("Print IR at pass with this number as "
+    PrintBeforePassNumber("print-before-pass-number", cl::init(0), cl::Hidden,
+                cl::desc("Print IR before the pass with this number as "
                          "reported by print-passes-names"));
```

Comment:
should this help text be `print-pass-numbers`?

Reply:
Fixed